docs/guides/universal-profile/browser-extension/01-install-browser-extension.md

```diff
@@ -12,130 +12,197 @@ import TabItem from '@theme/TabItem';
 The UP Browser Extenstion is currently in the **development alpha** version. DO NOT use this in production!
 :::
 
-In this guide, you will learn how to download + install the **Universal Profile browser extension** on Chrome, Edge, Opera or Brave.
+This guide will teach you how to download and install the Universal Profile browser extension on Chrome, Edge, Opera, or Brave.
 
-## Download + Install the extension
+## Download and Unpack the Extension
```

Comment:
why capitalise Extension and Unpack ?

Reply:
Sorry, leftover from wanting to split steps, as I described in the Network PR`s rules